numcodecs/tests/test_zlib.py

```diff
@@ -55,7 +55,7 @@ def test_repr():
 
 
 def test_alias():
```

Comment:
Can we not drop this test entirely if 'gzip' is no longer an alias for 'zlib'?

Reply:
Yes, I guess that would make sense.

Reply:
Definitely. That was the other route I was debating, but wanted you to confirm. 😄